operator/builtin/input/file/util_test.go

```diff
@@ -35,7 +35,7 @@ import (
 
 func newDefaultConfig(tempDir string) *InputConfig {
 	cfg := NewInputConfig("testfile")
-	cfg.PollInterval = helper.Duration{Duration: 50 * time.Millisecond}
```

Comment:
Is this change necessary?

Reply:
I think it being too short causes test to be flaky somehow. and 200ms is the default configuration anyway.

Reply:
let me try reverting this.